tslint-gci.json

```diff
@@ -1,22 +1,6 @@
 {
-	"extends": "./tslint.json",
+	"extends": ["./tslint.json", "./tslint.mssdl.required.json"],
```

Comment:
I don't think you need to separate this out into a different file. Matt created this file specifically for "mssdl", I think you can just not have it extend the original tslint and run the tslint targeting this file.

Reply:
We use the GCI config for the GCI Azure Pipeline so I'd like to leave that intact and doing the full check of all rules on the ADS rule base.

The best option (and one I'll be working on) is to have each repo have its own tslint config so we don't have this weird cross-repo dependency. But my primary concern here is to close out the TSA bugs asap and the quickest way to accomplish that is to continue using this one for now.

Reply:
What other repos are using this file?

Reply:
We also already do a full check for tslint, thats what the hygiene step does. If theres a different pipeline that we want this on, we should just add the hygiene step to it.

Reply:
So right now the Azure Pipelines we run as part of the GCI check just run yarn tslint, and so run the tslint check with the gci file. I don't see any problem with changing that to run gulp tslint instead though.

As for the other repos - it's basically how Matt (or whoever) set up the TSA scans. We have a single release that runs TSLint for ADS as well as all the sub-modules such as :

html-query-plan
sqlops-dataprotocolclient
...and so on

All of these currently using the same tslint config file (the one in the ADS repo). The TSA pipeline just downloads all the repos into the workspace and then runs tslint on each one in turn.

What I'll probably end up doing after thinking about it a bit more is just commit the tslint config to each repo separately and then run the scans using those - that way each repo has its own config file (as it should) and we can leave this one as-is.